ClientTests/UIImageViewExtensionsTests.swift

```diff
@@ -35,7 +35,13 @@ class UIImageViewExtensionsTests: XCTestCase {
     }
 
     func testAsyncSetIcon() {
-        let imageData = UIImagePNGRepresentation(UIImage(named: "fxLogo")!)
+        let originalWithMetadata = UIImage(named: "fxLogo")!
+        let originalFromPNG = UIImage(data: UIImagePNGRepresentation(originalWithMetadata)!)!
+        let imageData = UIImagePNGRepresentation(originalFromPNG)
+
+        // This confirms that the odd constructon step above is necessary
+        XCTAssert(UIImagePNGRepresentation(originalWithMetadata) != imageData,  "These differ in what appears to be the metadata block")
+
```

Comment:
@jhugman this test was failing when doing a byte-wise comparison of PNGs created from the bundle's UIImage and the network-loaded UIImage. The latter is constructed from a PNG.
My theory is the metadata block of the PNGs is different, because they appear visually identical in all cases. 
It seems accidental that the test was working without the steps I added here.